tests/pipeline/test_earnings.py

```diff
@@ -378,14 +382,22 @@ def test_infer_timestamp(self):
             infer_timestamps=True,
         )
         self.assertEqual(
-            loader.announcement_dates.keys(),
+            loader.events_by_sid.keys(),
             announcement_dates.keys(),
         )
         assert_series_equal(
-            loader.announcement_dates[0],
-            pd.Series(index=[dtx[0]] * 10, data=dtx),
+            loader.events_by_sid[0].loc[:, ANNOUNCEMENT_FIELD_NAME],
```

Comment:
this should just be standard getitem. I profiled and `df[colname]` is much faster than the loc. Sorry about the false information